### PR TITLE
[Feature] Adding new eslint rule sets + prettier

### PR DESCRIPTION
**Pull request by gsavchenko**
Just adding some popular/opinionated eslint rules to make it easier to deal with style nits in the frontend code. Should probably do this for the backend as well?

### CODE REVIEW

File: kur_kards_frontend/.eslintrc.json [hunk 1]
```diff
@@ -1,3 +1,13 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": ["next/core-web-vitals", "airbnb", "airbnb/hooks", "prettier"],
+  "rules": {
+    "import/extensions": ["error", "never"],
+    "react/jsx-filename-extension": [
+      1,
+      { "extensions": [".js", ".jsx", ".ts", ".tsx"] }
+    ],
+    "react/jsx-props-no-spreading": "off",
```
Comment (vicovaro) [inline kur_kards_frontend/.eslintrc.json:9]: YESSSSSS, I don't want to guess what's in the props anymore.

**Review by vicovaro — APPROVED** [review]
lgtm

File: kur_kards_frontend/.eslintrc.json (hunk 1 above)
Comment (gsavchenko) [reply kur_kards_frontend/.eslintrc.json:9]: i would like to point on that this rule is for turning off the rule so that we can do it

Comment (gsavchenko) [reply kur_kards_frontend/.eslintrc.json:9]: https://github.com/gsavchenko/kur-kards/blob/558f565d2dffe86edcc1cafa0febe19d4fbef4fe/kur_kards_frontend/pages/_app.tsx#L8

**Pull request merged**